hil-test/Cargo.toml

```diff
@@ -173,6 +173,15 @@ name              = "uart_async"
 harness           = false
 required-features = ["embassy"]
 
+[[test]]
+name              = "uart_uhci_async"
+harness           = false
+required-features = ["embassy"]
+
+[[test]]
+name              = "uart_uhci_blocking"
+harness           = false
```

Comment:
Merge these two test into one file. I think we avoid doing splits like these in general.